SWLOR.Game.Server/Feature/SpawnDefinition/SpaceSpawnDefinition.cs

```diff
@@ -327,7 +328,62 @@ private void KorribanOrbit()
         private void DathomirOrbit()
         {
             _builder.Create("SPACE_DATHOMIR_ORBIT", "Space - Dathomir Orbit")
-                
+
+                .AddSpawn(ObjectType.Creature, "pirate_drake")
```

Comment:
I'm confused about this change set. Why are there so many adjustments to Dathomir here?